### PR TITLE
Proxy setup

### PR DESCRIPTION
**Pull request by krisdo**

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -1,11 +1,17 @@
...
+const path = require('path');
 const app = express();
 const port = 2001;
 const db = require('../database-mongodb/index.js')
 
-app.use(express.static('./public'));
+app.use(express.static(path.join(__dirname, '../public')));
 app.use(express.json());
 app.use(express.urlencoded({extended: true}));
+app.use(function(req, res, next) {
```
Comment (cmonez) [inline server/app.js:10]: Im intrigued by whats going on here! I'm guessing you will list all the URLs in res.header  for the microservices your pulling from?

File: package.json [hunk 2]
```diff
@@ -7,6 +7,8 @@
   "engines": {
     "node": ">=6.13.0"
   },
+  "proxy": "http://localhost:1000/",
```
Comment (cmonez) [inline package.json:10]: Same question applies here.

File: server/app.js (hunk 1 above)
Comment (krisdo) [reply server/app.js:10]: For now I am, but it might change when we set everything up on AWS and/or set up env vars!

File: package.json (hunk 2 above)
Comment (krisdo) [reply package.json:10]: this one didn't work but I am leaving it here and going to ask during OH

File: server/app.js (hunk 1 above)
Comment (cmonez) [reply server/app.js:10]: oooohhhh. How exciting, nice work @krisdo !

**Pull request merged**